readme-docs/gather-docs.sh

```diff
@@ -0,0 +1,8 @@
+for f in $(find "./x" -type f -name "*.md" -not -path "deps/*" \
+            -not -path "scripts/*" \
+            -not -path "readme-docs/*"
```

Comment:
are the `-not`'s necessary since you're only looking in `./x`?

Reply:
nah not necessary, removed!